examples/chip-tool-darwin/templates/commands.zapt

```diff
@@ -211,6 +218,7 @@ public:
         AddArgument("attr-name", "{{asDelimitedCommand (asUpperCamelCase name)}}");
         AddArgument("min-interval", 0, UINT16_MAX, &mMinInterval);
         AddArgument("max-interval", 0, UINT16_MAX, &mMaxInterval);
+        AddArgument("fabric-filtered", 0, 1, &mFabricFiltered);
```

Comment:
Why add this to Darwin only, and not Linux as well?

This seems like this should not be Darwin-only

Reply:
`chip-tool` (which runs on both Linux and Darwin and uses the SDK C++ API directly) already supports these.

`chip-tool-darwin`, which runs on Darwin only and uses the Darwin framework API, not the SDK directly is what this PR is about.

Reply:
This is really confusing. These tools should be better named to express what they are more specifically

Reply:
Would renaming chip-tool-darwin to darwin-matter-tool help?

There's a tradeoff here between executable name length and expressiveness...

Reply:
@tcarmelveilleux  We have an [issue](https://github.com/project-chip/connectedhomeip/issues/14987) tracking the rename of this CLI. We haven't officially decided on a name.